previews/primer/alpha/nav_list_preview.rb

```diff
@@ -6,13 +6,11 @@ module Alpha
     class NavListPreview < ViewComponent::Preview
       # @label Playground
       def playground
-        render(Primer::Alpha::NavList.new(selected_item_id: :code_review_limits)) do |list|
-          list.with_group do |group|
-            group.with_heading(title: "Repository settings")
+        render(Primer::Alpha::NavList.new(aria: { label: "Repository settings" }, selected_item_id: :code_review_limits)) do |list|
+          list.with_heading(title: "Repository settings navigation")
```

Comment:
```suggestion
          list.with_heading(title: "Repository settings")
```

I think this copy was suggested, but it doesn't fit well as a large h2 tag to say "navigation" and I think the design and use of `nav` makes it clear.